matching/matcher.go

```diff
@@ -131,3 +132,35 @@ func filterPlacesOnPriceLevel(places []Place, level POI.PriceLevel) []Place {
 	}
 	return results
 }
+
+// Filter parameters related with user rating
+type UserRatingFilterParams struct {
+	MinUserRatings int
+}
+
+// Filter to remove low user rating
+type FilterLowUserRating struct {
```

Comment:
Consider rename to `MatcherForUserRatingsCount` to be consistent.

Reply:
Done.